### DISCUSSION TITLE
Does version 1.11.0 supports pgbouncer 1.7.2?

### DISCUSSION THREAD

**Discussion by kilroypr** [rapidloop/pgmetrics#41] [Q&A]
[piadmin@analytics ~]$ pgmetrics -V -p 5432 -h localhost pgbouncer
pgmetrics 1.11.0
[piadmin@analytics ~]$ pgmetrics -p 5432 -h localhost pgbouncer
Password:
pgmetrics: show pools query failed: sql: expected 11 destination arguments in Scan, not 12
[piadmin@analytics ~]$ psql -p 5432 -h localhost pgbouncer
Password for user piadmin:
psql (12.6, server 1.7.2/bouncer)
Type "help" for help.

pgbouncer=# show pools
pgbouncer-# ;
   database   |     user      | cl_active | cl_waiting | sv_active | sv_idle | sv_used | sv_tested | sv_login | maxwai
t |  pool_mode
--------------+---------------+-----------+------------+-----------+---------+---------+-----------+----------+-------
--+-------------
 milliman2    | mililman_test |         0 |          0 |         0 |       0 |       0 |         0 |        0 |
0 | transaction
 newanalytics | piadmin       |         0 |          0 |         0 |       0 |       0 |         0 |        0 |
0 | transaction
 pgbouncer    | pgbouncer     |         1 |          0 |         0 |       0 |       0 |         0 |        0 |
0 | statement
 postgres     | piadmin       |         0 |          0 |         0 |       0 |       0 |         0 |        0 |
0 | transaction
(4 rows)

pgbouncer=#

**Comment by mdevan** [comment]
It's missing the maxwait_us column that came in v1.8.  You can tweak the code in getPBPools() [here](https://github.com/rapidloop/pgmetrics/blob/master/collector/collect.go#L2334) to fix it.

v1.7 is over 5 years old, are you sure you don't want to upgrade instead?
